pallets/native-token-management/src/mock.rs

```diff
@@ -0,0 +1,92 @@
+use crate::mock::sp_runtime::testing::H256;
+use crate::DispatchResult;
+use crate::TokenTransferHandler;
+use frame_support::sp_runtime::traits::BlakeTwo256;
+use frame_support::sp_runtime::traits::IdentityLookup;
+use frame_support::sp_runtime::BuildStorage;
+use frame_support::traits::ConstU16;
+use frame_support::traits::ConstU32;
+use frame_support::traits::ConstU64;
```

Comment:
Cosmetic: I don't like wildcard imports, to be honest, but okay. However, "paid by LOC" style of imports is even worse.